lib/bolt/error.rb

```diff
@@ -24,6 +24,10 @@ def to_h
       h
     end
 
+    def add_filelineno(details)
+      @details.merge!(details) unless @details['file']
+    end
```

Comment:
It might be helpful to document that plans will add `file` and `line` to `details` unless `file` is already defined.

Reply:
I dunno - to me it feels like expected behavior and kind of clutters the docs, but I added it for consideration.